_data/languages.yml

```diff
@@ -1,30 +1,1065 @@
+  typology:
+    - SVO
+    - inflected
+  variants: [ br, pt ]
+
+-
+  code: ro
+  name: Romanian
+  family: rum
+  scripts: [ Cyrl, Latn ]
+  typology:
+    - SVO
+    - synthetic
+  variants: [ md, ro ]
+
+-
+  code: ru
+  name: Russian
+  family: sla
+  scripts: [ Cyrl ]
+  typology:
+    - SVO
+    - inflected, fusional, synthetic
+  variants: [ ru, md, kz, ua ]
+
+-
+  code: rw
+  name: Kinyarwanda
+  family: bnt
+  scripts: [ Latn ]
+  typology:
+    - SVO
+    -
+  variants: [ rw ]
+
+-
+  code: sd
+  name: Sindhi
+  family: snd
+  scripts: [ Arab, Deva, Khoj, Sind ]
+  typology:
+    - SOV
+    - inflected
+  variants:
+
+-
+  code: si
+  name: Sinhala
+  family: ine
+  scripts: [ Sinh ]
+  typology:
+    - SOV
+    - fusional
+  variants: [ lk ]
+
+-
+  code: sk
+  name: Slovak
+  family: ine
+  scripts: [ Latn ]
+  typology:
+    - SVO
+    - synthetic
+  variants: [ sk ]
+
+-
+  code: sl
+  name: Slovenian
+  family: ine
+  scripts: [ Latn ]
+  typology:
+    - SOV
+    - fusional, inflected
+  variants: [ si ]
+
+-
+  code: sm
+  name: Samoan
+  family: map
+  scripts: [ Latn ]
+  typology:
+    - VSO
+    - analytic
+  variants: [ ws ]
+
+-
+  code: sn
+  name: Shona
+  family: nic
+  scripts: [ Latn ]
+  typology:
+    - SVO
+    - agglutinative
+  variants:
+
+-
+  code: so
+  name: Somalia
+  family: afa
+  scripts: [ Arab, Latn, Osma ]
+  typology:
+    - SOV
+    - agglutinative
+  variants: [ dj, et, ke, so ]
+
+-
+  code: sq
+  name: Albanian
+  family: sqj
+  scripts: [ Latn, Elba ]
+  typology:
+    - SVO
+    - fusional
+  variants: [ al ]
+
+-
+  code: sr
+  name: Serbian
+  family: ine
+  scripts: [ Cyrl, Latn ]
+  typology:
+    - SVO
+    - synthetic
+  variants: [ ba, me, rs ]
+
+-
+  code: st
+  name: Southern Sotho
+  family: bnt
+  scripts: [ Latn ]
+  typology:
+    - SVO
+    - agglutinative
+  variants: [ za ]
+
+-
+  code: su
+  name: Sundanese
+  family: map
+  scripts: [ Latn, Sund ]
+  typology:
+    - SVO
+    -
+  variants:
+
+-
+  code: sv
+  name: Swedish
+  family: ine
+  scripts: [ Latn ]
+  typology:
+    - SVO
+    - inflected
+  variants: [ fi, se ]
+
+-
+  code: sw
+  name: Swahili
+  family: nic
+  scripts: [ Latn ]
+  typology:
+    - SVO
+    - agglutinative
+  variants: [ cd, ke, tz, ug ]
+
+-
+  code: ta
+  name: Tamil
+  family: dra
+  scripts: [ Taml ]
+  typology:
+    - SOV
+    - agglutinative
+  variants: [ in, lk, my, sg ]
+
+-
+  code: te
+  name: Telugu
+  family: dra
+  scripts: [ Telu ]
+  typology:
+    - SOV
+    - agglutinative
+  variants: [ in ]
+
+-
+  code: tg
+  name: Tajik
+  family: iir
+  scripts: [ Arab, Cyrl, Latn ]
+  typology:
+    - SOV
+    - analytic
+  variants: [ tj ]
+
+-
+  code: th
+  name: Thai
+  family: tai
+  scripts: [ Thai ]
+  typology:
+    - SVO
+    - analytic, isolating
+  variants: [ th ]
+
+-
+  code: tk
+  name: Turkmen
+  family: trk
+  scripts: [ Arab, Cyrl, Latn ]
+  typology:
+    - SOV
+    - agglutinative
+  variants: [ iq ]
```

Comment:
tm, ir, af

Iraqi/Syrian Turkmen is not really Turkmen in the linguistic sense, more just Turkish.